package.json

```
@@ -2,7 +2,7 @@
   "name": "@chanzuckerberg/eds",
   "version": "12.0.0",
   "description": "The React-powered design system library for Chan Zuckerberg Initiative education web applications",
-  "author": "CZI <edu-frontend-infra@chanzuckerberg.com>",
+  "author": "CZI <edu-dev-foundations@chanzuckerberg.com>",
```

Checked and this does not exist yet.

```suggestion
  "author": "CZI <edu-frontend-infra@chanzuckerberg.com>",
```

---

@jeremiah-clothier when logged in, I hit groups.google.com and searched for the team alias, but it wasn't defined. I'll revert using that suggestion above